spec/lib/auth0/client_spec.rb

```diff
@@ -84,13 +83,12 @@
       it_should_behave_like 'Authentication API client'
     end
 
-    context 'with token, audience, and client_secret' do
+    context 'with token and client_secret' do
       let(:subject) do
         Auth0::Client.new(
           token: access_token,
           domain: domain,
           client_secret: client_secret,
-          audience: audience
```

Comment:
The audience parameter was used in the test, but as far as I investigated, this option has never been used for anything before.